archive/imo/imo1972_b2.lean

```diff
@@ -0,0 +1,122 @@
+/-
+Copyright (c) 2020 Ruben Van de Velde, Stanislas Polu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Ruben Van de Velde, Stanislas Polu
+-/
+
+import data.real.basic
+import data.set.basic
+import analysis.normed_space.basic
+
+/-- IMO 1972 B2
+
```

Comment:
It'd be good to provide a statement of the problem here.

Reply:
Done 👍